bin/hunter_env.sh

```diff
@@ -39,7 +39,9 @@ git submodule update --init --recursive --quiet
 uname -a
 
 # Install Python 3
-if [[ "$(uname)" == "Darwin" ]] && [ ! $(which python3) ]; then travis_retry brew install python3; fi
+# >> Error: python 2.7.14 is already installed \n To upgrade to 3.6.4_3, run `brew upgrade python`
+# if [[ "$(uname)" == "Darwin" ]] && [ ! $(which python3) ]; then travis_retry brew install python3; fi
```

Comment:
See https://github.com/ingenue/hunter/blob/3834d1dff2af8424b65887b51f7ffdfab3870c01/.travis.yml#L104-L106

Reply:
i'll update it.  thanks.